configs/projects/bolt-runtime.rb

```diff
@@ -199,13 +199,15 @@
   proj.component 'rubygem-yard'
 
   # Core Windows dependencies
-  proj.component 'rubygem-win32-dir'
-  proj.component 'rubygem-win32-process'
-  proj.component 'rubygem-win32-security'
-  proj.component 'rubygem-win32-service'
-  proj.component 'rubygem-windows_error'
-  proj.component 'rubygem-winrm'
-  proj.component 'rubygem-winrm-fs'
+  if platform.is_windows?
+    proj.component 'rubygem-win32-dir'
+    proj.component 'rubygem-win32-process'
+    proj.component 'rubygem-win32-security'
+    proj.component 'rubygem-win32-service'
+    proj.component 'rubygem-windows_error'
+    proj.component 'rubygem-winrm'
+    proj.component 'rubygem-winrm-fs'
+  end
```

Comment:
This is a little surprising, but we only want to omit `rubygem-win32-*` on non-Windows. But we do want `rubygem-winrm*` on all hosts so that Windows and non-Windows hosts can connect to and execute commands on remote Windows hosts. I think `rubygem-windows_error` is a dependency of `rubygem-ruby_smb` (defined earlier) need the same treatment, since bolt uses them to transfer files between `* <-> Windows`.

Reply:
Thanks for the quick review and explanation, I've updated my commit to only omit the `rubygem-win32-*` gems.